src/settings.ts

```diff
@@ -79,7 +79,7 @@ export function getEnvVar(varName: string, asType: "stringArray"): undefined | s
 /** Grabs an environment variable's value, and casts it to a `number[]` */
 export function getEnvVar(varName: string, asType: "numberArray"): undefined | number[]
 /** Grabs an environment variable's value, and casts it to a specific type (default string) */
-export function getEnvVar<T extends ("string" | "number" | "stringArray" | "numberArray")>(varName: string, asType: T = "string" as T): undefined | (string | number | string[] | number[])
+export function getEnvVar<T extends ("string" | "number" | "stringArray" | "numberArray" | "stringNoEmpty")>(varName: string, asType: T = "string" as T): undefined | (string | number | string[] | number[])
```

Comment:
WUT

Reply:
I don't know mane. It was bitching about that not being an option so I added it. It's Sv magic code so I don't touch it

Reply:
it no longer bitches but im still gonna keep the change cuz its broken in TS 5.2

Reply:
yea I actually knew that was a bug, just not that it would actually error